### ISSUE TITLE
scss/operator-no-unspaced conflicts with Tailwind CSS children styling

### ISSUE THREAD

**Issue by karlhorky** [stylelint-scss/stylelint-scss#999]
Hi there, thanks for `stylelint-scss`!

Using [Tailwind CSS Direct Children Styling](https://tailwindcss.com/docs/hover-focus-and-other-states#styling-direct-children) using `@apply` and classes with the `*:` prefix leads to a problem being reported by `scss/operator-no-unspaced`:

```css
@layer components {
  .fullWidthChildren {
    @apply *:w-full; /* 💥 Expected single space after "*"  scss/operator-no-unspaced */
  }
}
```

Would there be any way around this without disabling the rule? Ideally I'd like to leave the rule turned on...

**Comment by kristerkari** [comment]
Thanks @karlhorky . This is definitely a bug. 

You can check stylelint docs on how to disable a rule using comments:
https://stylelint.io/user-guide/ignore-code/

**Comment by karlhorky** [comment]
Thanks for the PR and release in [`stylelint-scss@6.3.1`](https://github.com/stylelint-scss/stylelint-scss/releases/tag/v6.3.1)! 🙌

**Comment by kristerkari** [comment]
Thanks for reporting the bug @karlhorky. Please let me know if you find more things that are not working. 👍
